app/src/main/java/org/mozilla/rocket/content/ContentPortalView.kt

```diff
@@ -50,6 +44,8 @@ class ContentPortalView : CoordinatorLayout, ContentPortalListener {
     private var newsAdapter: NewsAdapter<NewsItem>? = null
     private var newsListLayoutManager: LinearLayoutManager? = null
 
+    private val contentFeatureUseCase = ContentFeaturePresenter()
```

Comment:
`ContentFeaturePresenter` should rename to `ContentFeatureUseCase` or `ContentFeatureModel` since its role is not like what presenter do.

Reply:
I agree it's not a presenter. It didn't interact with the view and model. And firstly I also name it ContentFeatureUseCase. But at the second thought, it's not like an Interactor/UseCase either. It doesn't interact with the presenter and the controller. It looks much like a model but since I didn't have any architecture here, I don't want to call it model. I'll just use the name `ContentFeature`, and change the name if required in the future.

Reply:
Thanks for the detail.